src/content/1.7/modules/concepts/templating/vuejs/_index.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Using Vue.js
+menuTitle: Vue.js
+weight: 2
+chapter: true
+---
+
+# Using Vue.js
+
+Depending on the PrestaShop version running on the shop, either on the back or the front-office,
+at least one template renderer is available to display content on the page. With Javascript,
+it is possible to update & animate the page content without having to refresh the page.
+
+jQuery is available on PrestaShop to help implementing these interface updates.
+For complex interfaces, with state management for instance, other frontend options could be considered.
+[Vue.js](https://vuejs.org/) is a JavaScript Framework that will be helpful in this case.
+This chapter covers its integration on a module.
+
+{{% children %}}
+
+## 1- Creation of a Vue.js application in the module directories
+
+The objective of this part is to make Vue.js available in the project. 
+
+We will store the Vue.js source code in a dedicated folder that will be present in the module only
+during the development part. Running Vue.js will be possible by compiling and storing the JS files
+in the proper assets folder.
+
+During this tutorial, we use a new folder `_dev` that will be created at the root directory of the module.
+
+Installation of Vue.js can be done via [several ways](https://vuejs.org/v2/guide/installation.html),
+but we will use the CLI option to get all the stack.
+
+* Installation of Vue.js CLI tool (globally)
+
+```
+$ npm install -g @vue/cli
+```
+
+* Creation of the project
+
+```
+$ vue create _dev
+```
+
+We chose the preset with Vue 2 & Babel for this tutorial.
+
+After the initialization process with some questions and downloads, a whole standalone application
+will be created and available in the `_dev` folder. Running npm commands are defined in the `package.json`
+file to run or build it.
+
+```
+$ cd _dev
+$ npm run build
+```
+
+At the moment, we have a Vue.js application that can run independantly from the module.
+In the next parts, this standalone application will be adapted to run inside a PrestaShop page.
```

Comment:
```suggestion
In the next parts, this standalone application will be adapted to run inside a PrestaShop page.

```

Looks like you're using an empty lien everywhere else before a title

Reply:
Indeed!